identifiers/country-in/vidhan_parishad.csv

```diff
@@ -0,0 +1,215 @@
+ocd-division/country:in/state:ap/special_constituency:graduate/ed:kurnool_kadapa_anantapur,Kurnool-Kadapa-Anantapur
+ocd-division/country:in/state:ap/special_constituency:graduate/ed:prakasam_nellore_chittoor,Prakasam-Nellore-Chittoor
+ocd-division/country:in/state:ap/special_constituency:graduate/ed:vishakapatnam_vizianagaram_srikakulam,Vishakapatnam-Vizianagaram-Srikakulam
+ocd-division/country:in/state:ap/special_constituency:local_authority,Local body special constituency
+ocd-division/country:in/state:ap/special_constituency:local_authority/ed:anantapur,Anantapur
+ocd-division/country:in/state:ap/special_constituency:local_authority/ed:chittoor,Chittoor
+ocd-division/country:in/state:ap/special_constituency:local_authority/ed:east_godavari,East Godavari
+ocd-division/country:in/state:ap/special_constituency:local_authority/ed:guntur,Guntur
+ocd-division/country:in/state:ap/special_constituency:local_authority/ed:kadapa,Kadapa
+ocd-division/country:in/state:ap/special_constituency:local_authority/ed:kurnool,Kurnool
+ocd-division/country:in/state:ap/special_constituency:local_authority/ed:nellore,Nellore
+ocd-division/country:in/state:ap/special_constituency:local_authority/ed:prakasam,Prakasam
+ocd-division/country:in/state:ap/special_constituency:local_authority/ed:srikakulam,Srikakulam
+ocd-division/country:in/state:ap/special_constituency:local_authority/ed:vishakapatnam,Vishakapatnam
+ocd-division/country:in/state:ap/special_constituency:local_authority/ed:vizianagaram,Vizianagaram
+ocd-division/country:in/state:ap/special_constituency:local_authority/ed:west_godavari,West Godavari
+ocd-division/country:in/state:ap/special_constituency:teacher,Teacher special constituency
+ocd-division/country:in/state:ap/special_constituency:teacher/ed:anantapur_kadapa_kurnool,Anantapur-Kadapa-Kurnool
+ocd-division/country:in/state:ap/special_constituency:teacher/ed:chittoor_nellore_prakasam,Chittoor-Nellore-Prakasam
+ocd-division/country:in/state:ap/special_constituency:teacher/ed:east_godavari_west_godavari,East Godavari-West Godavari
+ocd-division/country:in/state:ap/special_constituency:teacher/ed:guntur_krishna,Guntur-Krishna
+ocd-division/country:in/state:ap/special_constituency:teacher/ed:srikakulam_vizianagaram_vishakapatnam,Srikakulam-Vizianagaram-Vishakapatnam
+ocd-division/country:in/state:br/special_constituency:graduate,Graduate special constituency
+ocd-division/country:in/state:br/special_constituency:graduate/ed:darbhanga,Darbhanga
+ocd-division/country:in/state:br/special_constituency:graduate/ed:gaya,Gaya
+ocd-division/country:in/state:br/special_constituency:graduate/ed:koshi,Koshi
+ocd-division/country:in/state:br/special_constituency:graduate/ed:patna,Patna
+ocd-division/country:in/state:br/special_constituency:graduate/ed:saran,Saran
+ocd-division/country:in/state:br/special_constituency:graduate/ed:tirhut,Tirhut
+ocd-division/country:in/state:br/special_constituency:local_body,Local body special constituency

```

Comment:
Shouldn't this be `local_authority` like the others?

Reply:
Bihar uses `local_body` to refer to this division instead, and so we are following the state / ECI to be consistent.

Reply:
Thanks for clarifying, consistency with the local naming sounds good to me.